### ISSUE TITLE
[INFO] HomeAssistant minimum requirement is version 2021.9.x

### ISSUE THREAD

**Issue by CipherLab** [ollo69/ha-smartthinq-sensors#222]
**Describe the bug**
Everything goes smoothly when I add integration via hacs, and then the integration page, and follow the url's. But when the integration loads I get errors that the sensors could not be loaded

**Environment details:**
 - Environment (Haasio - Docker)
 - Home Assistant version installed: Home Assistant 2021.6.6
 - Component version installed: v0.10.1

**Output of HA logs**
Paste the relavant output of the HA log here.

```

Unable to prepare setup for platform smartthinq_sensors.sensor: Platform not found (cannot import name 'DEVICE_CLASS_PM1' from 'homeassistant.components.sensor' (/usr/src/homeassistant/homeassistant/components/sensor/__init__.py)).
Unable to prepare setup for platform smartthinq_sensors.binary_sensor: Platform not found (cannot import name 'BinarySensorEntityDescription' from 'homeassistant.components.binary_sensor' (/usr/src/homeassistant/homeassistant/components/binary_sensor/__init__.py)).
Unable to prepare setup for platform smartthinq_sensors.climate: Platform not found (cannot import name 'ClimateEntityDescription' from 'homeassistant.components.climate' (/usr/src/homeassistant/homeassistant/components/climate/__init__.py)).
Unable to prepare setup for platform smartthinq_sensors.switch: Platform not found (cannot import name 'SwitchEntityDescription' from 'homeassistant.components.switch' (/usr/src/homeassistant/homeassistant/components/switch/__init__.py)).

```

**Comment by ollo69** [comment]
+1 (I will start count the number of time this kind of issue was opened).
Fix is very simple, upgrade HA at least to 2021.9.x as specified in [release note](https://github.com/ollo69/ha-smartthinq-sensors/releases).

**Comment by ollo69** [comment]
Closing this, now integration provide proper message about requirement,
